mat64/list_test.go

```diff
@@ -0,0 +1,684 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mat64
+
+import (
+	"fmt"
+	"math/rand"
+	"reflect"
+
+	"github.com/gonum/blas/blas64"
+	"github.com/gonum/floats"
+	"gopkg.in/check.v1"
+)
+
+// The sizes are legal when the two matrices have the same rectangular shape.
```

Comment:
Make these func comments follow the doc comment style, "dimsOKSameRectangular returns whether ...".